### PR TITLE
Githubstudent denver

### PR DESCRIPTION
**Pull request by dnrdevops**
----
@githubteacher

### CODE REVIEW

**Comment by aditya21891** [comment]
dnrdevops . Good to learn git with you. Great work. Hope this will help you.

**Comment by lindsijoyce** [comment]
@dnrdevops make sure your github handle is your username, not your email address.

File: _pins/dnrdevops@outlook.com.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: dnrdevops@outlook.com
```
Comment (crichID) [inline _pins/dnrdevops@outlook.com.json]: @dnrdevops your githubHandle should just be your user name, not your email address :wink:

**Comment by crichID** [comment]
@dnrdevops you will want to remove `@outlook.com` from your file name! If you need help, please check out this video: https://vimeo.com/201011288

**Comment by hollenberry** [comment]
Hey @guopei0001 ! Looks like you might be editing the wrong branch! Please create a new one so that @dnrdevops can try and make his fixes.

**Comment by hollenberry** [comment]
@dnrdevops  I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/6603/commits/a5df20ed04beed0bcd5c9afe04d2044b355815cd. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by May 26, I will go ahead and do it just to keep the repository clean!

**Pull request merged**